dune

```diff
@@ -4,7 +4,9 @@
  (release (flags :standard -rectypes)
           (ocamlopt_flags -O3 -unbox-closures))
  (ireport (flags :standard -rectypes -w -9-27-50+40+60)
-          (ocamlopt_flags :standard -O3 -unbox-closures -inlining-report)))
+          (ocamlopt_flags :standard -O3 -unbox-closures -inlining-report))
+ (ocaml408
+  (flags :standard -strict-sequence -strict-formats -short-paths -keep-locs -rectypes -w -9-27-50+40+60 -warn-error -5 -alert --deprecated)))
```

Comment:
What does `-alert --deprecated` do?

Reply:
Alerts are a new feature in 4.08 , see OCaml's changelog.